examples/tools/gamepad_viewer.rs

```diff
@@ -393,12 +393,12 @@ fn update_buttons(
     materials: Res<ButtonMaterials>,
     mut query: Query<(&mut MeshMaterial2d<ColorMaterial>, &ReactTo)>,
 ) {
-    for gamepad in &gamepads {
+    for buttons in &gamepads {
```

Comment:
I think that `gamepad` is much clearer here.

Reply:
Agreed!